songbank-backend/backend.js

```diff
@@ -88,28 +85,20 @@ app.get('/auth/search/:id', (req, res) => {
             'Authorization' :  `Bearer ${access_token}`
         },
     };
-    console.log('MADE IT HERE');
-
     request.get(searchOptions, function(error, response, body) {
         if (!error && response.statusCode === 200) {
-            //access_token = //body.access_token;
-            //console.log(' :) ' + body);
             songTitle = body;
             res.send(songTitle);
-            console.log("hello: " + songTitle);
-            //res.redirect('/');
         }

```

Comment:
Removing comments and unused code like this makes the code much cleaner good job!